packages/Search/src/details/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+SET(HEADERS "")
+SET(SOURCES "")
+
+TRIBITS_CONFIGURE_FILE(${PACKAGE_NAME}_config.h)
+
+SET_AND_INC_DIRS(DIR ${CMAKE_CURRENT_SOURCE_DIR})
+APPEND_GLOB(HEADERS ${DIR}/*.h)
+APPEND_GLOB(HEADERS ${DIR}/*.hpp)
+APPEND_GLOB(SOURCES ${DIR}/*.cpp)
+TRILINOS_CREATE_CLIENT_TEMPLATE_HEADERS(${DIR})
+
+# Must glob the binary dir last to get all of the auto-generated headers
+SET_AND_INC_DIRS(DIR ${CMAKE_CURRENT_BINARY_DIR})
+APPEND_GLOB(HEADERS ${DIR}/*.hpp)
+APPEND_SET(HEADERS ${DIR}/${PACKAGE_NAME}_config.h)
+APPEND_SET(HEADERS ${DIR}/${PACKAGE_NAME}_ETIHelperMacros.h)
```

Comment:
Can we replace this by `${CMAKE_CURRENT_BINARY_DIR}/../${PACKAGE_NAME}_ETIHelperMacros.h`?